frontend/src/v5/validation/auth.ts

```diff
@@ -16,11 +16,41 @@
  */
 
 import * as Yup from 'yup';
+import { formatMessage } from '../services/intl';
 
-const username = Yup.string().required();
+export const username = Yup.string().required();
 const password = Yup.string().required();
 
 export const LoginSchema = Yup.object().shape({
 	password,
 	username,
 });
+
+export const PasswordForgotSchema = Yup.object().shape({
+	username,
+});
+
+export const PasswordChangeSchema = Yup.object().shape({
+	newPassword: Yup.string()
+		.required(
+			formatMessage({
+				id: 'passwordChange.error.passwordRequired',
+				defaultMessage: 'Password is required',
+			}),
+		)
+		.max(65,
+			formatMessage({
+				id: 'passwordChange.error.tooLong',
+				defaultMessage: 'Password is limited to 65 characters',
+			})),
```

Comment:
I believe the password validation schema should also check the password strength. This is what I wrote for the registration form: https://github.com/3drepo/3drepo.io/pull/3149/files#diff-96fc3af36608ace028221cd1949c5fb813cf03dc95a25b58a72c84e3b9743880

Reply:
I've copied the password requirements from the V5 backend (https://github.com/3drepo/3drepo.io/blob/ISSUE_3112/backend/src/v5/utils/helper/yup.js)

As for the password strength. I let the backend handle the password and if it's too weak (or any other errors occur) I use the error messages from theere. I'm not sure if my method is best practise so I'll ask Santiago about it

Reply:
but to be clear, do it on both api and frontend as we did have a previous case of the api not honouring the password strength https://github.com/3drepo/3drepo.io/issues/2453

Reply:
also pet peeve I see happen a lot with the client side password validation is that it kicks in as soon as people start typing putting ugly red boxes up and possibly moving/redrawing elements and it's super annoying. The validation should kick in on mouse out/stopped typing for a few seconds to make sure the users ceased input.